src/cli.ts

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env node
```

Comment:
We could use a shebang like [this](https://github.com/pi0/pipress/blob/bd8011823c54dec9d034a2ef93281e77c86d7fc5/server/index.mjs#L1-L2) to be more inclusive.